readalongs/text/make_package.py

```diff
@@ -20,13 +20,17 @@
 
 from lxml import etree
 
-from readalongs._version import VERSION
+from readalongs._version import CURRENT_WEB_APP_VERSION, VERSION
```

Comment:
nice idea, moving `CURRENT_WEB_APP_VERSION` to `_version.py` and reusing it here.